lit/src/app.ts

```diff
@@ -92,11 +92,15 @@ export class App extends MobxLitElement {
       case Pages.PARTICIPANT:
         return this.renderParticipantView(this.renderParticipant());
       case Pages.PARTICIPANT_STAGE:
-        // TODO: Ignore this for experimenters, and allow experimenters to add this.
-        if (false) {
+        if (
+          !this.authService.isExperimenter &&
```

Comment:
What do you think about verifying "isExperimenter" (and maybe the attention check params?) inside the attention-check-popup component (so that it's not possible to render it for experimenters viewing things)? Just to specifically safeguard the popup against experimenters in preview mode (since we could use it in other locations later).

Reply:
We can follow up with this refactor though!

Reply:
Done!